src/app/components/AnalyticsConsent/initializeMatomo.ts

```diff
@@ -0,0 +1,53 @@
+import { redactURL } from './redact'
+
+declare global {
+  interface Window {
+    _paq: [string, ...any[]][]
+  }
+}
+
+// Generate snippet https://matomo.oasis.io/index.php?module=CoreAdminHome&action=trackingCodeGenerator&idSite=1&period=day&date=yesterday&updated=false
+// Snippet from https://developer.matomo.org/guides/tracking-javascript-guide#finding-the-piwik-tracking-code
+window._paq = window._paq || []
+export const matomoDomain = 'https://matomo.oasis.io/'
```

Comment:
New issue: pontusx.
I would just enable/disable our analytics with .env.production (instead of generalizing everything in env: matomoDomain, setDomains, setCookieDomain, setSiteId, sha384-)

Reply:
what's .env.production?

Reply:
https://github.com/oasisprotocol/explorer/blob/master/.env.production